### PR TITLE
feat: add restart command for tmuxinator

### PR DESCRIPTION
**Pull request by cdmoss**
A first draft to address #1553. Adds a tmuxinator alias `restart` that runs the new script "restart-tmux", which sends the commands referenced in this [message](https://discord.com/channels/990354215060795454/1060246795004944466/1064727278568878160) and runs gw-reload.sh. Not sure if this is the best approach so I definitely welcome suggested alternatives.

### CODE REVIEW

**Comment by cdmoss** [comment]
I just noticed it accidentally deleted one of the old aliases in the tmuxinator.yml file, I'll fix that right away.

**Comment by cdmoss** [comment]
I just noticed I deleted one of the original aliases in .tmuxinator.yml, I'll fix that right away.

File: scripts/restart-tmux.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+ 
+tmux send-keys -t fedimint-dev:1.4 "cargo build --bin fedimintd" C-m
+tmux send-keys -t fedimint-dev:1.4 "pkill -9 fedimintd" C-m
```
Comment (elsirion) [inline scripts/restart-tmux.sh:4]: Is that the right way around? This mixes federation and build logs, so if the build fails new epoch logs might hide that fact.

**Review by elsirion — CHANGES_REQUESTED** [review]

**Comment by elsirion** [comment]
Needs rebase.

File: scripts/restart-tmux.sh (hunk 1 above)
Comment (cdmoss) [reply scripts/restart-tmux.sh]: Point taken on the ordering, I switched it around. I'm not sure if the convenience tradeoff from using -9 is worth it, so I'll defer to your judgement on that - I can certainly change it to something more graceful.

Comment (elsirion) [reply scripts/restart-tmux.sh]: :+1: Let's leave it till graceful shutdown is faster (fix some stuck tasks)

**Comment by elsirion** [comment]
Can you rebase on the current master please to remove the merge conflict?

File: .tmuxinator.yml [hunk 2]
```diff
@@ -10,6 +10,8 @@ pre_window:
   - alias gateway-cli="\$FM_GATEWAY_CLI"
   - alias mint_rpc_client="\$FM_MINT_RPC_CLIENT"
   - alias fedimint-dbdump="\$FM_DB_DUMP"
+  - alias restart="./scripts/restart-tmux.sh"
+  - alias dbtool="\$FM_DB_TOOL"
```
Comment (elsirion) [inline .tmuxinator.yml:14]: This line seems unrelated? Did it get added by a rebase gone wrong?

**Review by elsirion — APPROVED** [review]

File: .tmuxinator.yml (hunk 2 above)
Comment (cdmoss) [reply .tmuxinator.yml:12]: I accidentally deleted the dbtool line early on, and added it back in with a commit that I squashed in the rebase. I thought I had rebased properly, but I could have messed something up, I'm kind of new to maintaining a tidy history... Thanks for approval! :)
